deploy/examples/create-external-cluster-resources.py

```diff
@@ -775,7 +775,9 @@ def get_active_and_standby_mgrs(self):
                 json_out.get("mgrmap", {}).get("services", {}).get("prometheus", "")
             )
             if not monitoring_endpoint:
-                return "", ""
+                raise ExecutionFailureException(
+                    f"can't find monitoring_endpoint, prometheus module might not be enabled"
```

Comment:
Can you also add a command to enable it in the error? That would be helpful for user IMO.

Reply:
Could there be other reasons that the endpoint would not be returned? If we tell them to enable the module but it's already enabled, that would also be confusing.

Reply:
We added it to the script as a precaution since we hit the issue, we can consult Ceph developers for insights if there can be any other reason.

Reply:
Added,
We read `ceph status` output, in the `service`  column if it is not present so most likely it is not bein enabled